modules/graph/CMakeLists.txt

```diff
@@ -43,6 +43,14 @@ target_link_libraries(vineyard_graph vineyard_client
                                      ${ARROW_SHARED_LIB}
                                      ${MPI_CXX_LIBRARIES}
 )
+
+# build vineyard-bt
+if(${LIBUNWIND_FOUND})
+    file(GLOB_RECURSE BT_SRC_FILES "${PROJECT_SOURCE_DIR}/src/common/backtrace/*.cc")
+    add_library(vineyard_bt ${BT_SRC_FILES})
+    target_link_libraries(vineyard_bt PRIVATE ${LIBUNWIND_LIBRARIES})
+endif()
```

Comment:
That is bad, since `"${PROJECT_SOURCE_DIR}/src/common/backtrace/*.cc"` will be included in many targets.